README.md

```diff
@@ -2,55 +2,103 @@
 
 [![Travis Build Status](https://travis-ci.org/oktal/pistache.svg?branch=master)](https://travis-ci.org/oktal/pistache)
 
-Pistache is a modern and elegant HTTP and REST framework for C++.
+Pistache is a modern and elegant HTTP and REST framework for C++. It is entirely written in pure-C++11 and provides a clear and pleasant API. 
 
-It is entirely written in pure-C++11 and provides a clear and pleasant API
+# Documentation
 
-Full documentation is located at [http://pistache.io](http://pistache.io).
+We are still looking for a volunteer to document fully the API. In the mean time, partial documentation is available at [http://pistache.io](http://pistache.io). If you are interested in helping with this, please open an issue ticket.
 
 # Contributing
 
-Pistache is an open-source project and will always stay open-source.  Contributors are welcome!
```

Comment:
I do not agree with this.  Its polarizing.

Reply:
That's effectively what was in the file before I patched it. I personally would prefer it *free* as in freedom instead of open source. But this isn't critical.

Reply:
Oh I see what you mean. Sorry, I misread your post. How would you prefer it worded?

Reply:
I do not want to debate OSS vs FSF here.  Pistache is licensed under Apache 2.0.  If you feel strongly that you need to remove the OSS reference that Oktal originally used, then replace it with a 1-liner about the Apache license.

Reply:
Fixed.